File: marketplace/adoptium-marketplace-schema/src/main/java/net/adoptium/marketplace/schema/ReleaseType.java
package net.adoptium.marketplace.schema;

import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
import org.eclipse.microprofile.openapi.annotations.media.Schema;

@Schema(type = SchemaType.STRING, defaultValue = "ga", enumeration = {"ga", "ea"})
public enum ReleaseType {
    ga,
    ea;
}

File: adoptium-models-parent/adoptium-api-v3-models/src/main/kotlin/net/adoptium/api/v3/models/ReleaseType.kt
package net.adoptium.api.v3.models

import org.eclipse.microprofile.openapi.annotations.enums.SchemaType
import org.eclipse.microprofile.openapi.annotations.media.Schema

@Schema(type = SchemaType.STRING, defaultValue = "ga", enumeration = ["all", "ga", "ea"])
enum class ReleaseType {
    all,
    ga,
    ea;
}
